### PR TITLE
npm: Pin versions for node-8

### PR DESCRIPTION
**Pull request by rzr**
Node-10 to be then supported

Origin: https://github.com/noble/noble/pull/851/commits/e2c476ec0c44d457dca22abf698f03787509c7ed
Forwarded: https://github.com/noble/noble/pull/851
Change-Id: I3c20732e71456cbe2b33509ccf2e0b72ca75442f
Signed-off-by: Philippe Coval <p.coval@samsung.com>

### CODE REVIEW

**Comment by mrstegeman** [comment]
Old Node versions should probably be dropped from the CI configs.

**Comment by Apollon77** [comment]
The minimum approach is only to remove 0.8 because failing in tests, other idea would be to go as in bluetooth-hci-socket and test 6, 8, 10

Appveyor needs a test config file at all because not existing so far

**Comment by rzr** [comment]
yes PR welcome on this repo too. To validate our abilty to co maintain it (forward upstream first).
I can explain more if needed, or look at this flow:
https://www.slideshare.net/rzrfreefr/tizen-upstreamcooptdc2014pcoval/5

So please disable obsolete node versions also in upstream then I will pick patch here too

https://travis-ci.org/noble/noble/builds/469648785?utm_source=github_status&utm_medium=notification

File: package.json [hunk 1]
```diff
@@ -33,20 +33,20 @@
     "win32"
   ],
   "dependencies": {
-    "debug": "~2.2.0"
+    "debug": "^4.1.0"
   },
   "optionalDependencies": {
-    "bluetooth-hci-socket": "^0.5.1",
+    "@abandonware/bluetooth-hci-socket": "^0.5.3-0",
```
[the remark this answers is not in the document]
Comment (rzr) [reply package.json]: logs welcome

**Comment by rgerr** [comment]
Any idea when node v10 will be supported for Noble?

**Comment by rzr** [comment]
@rgerr  try master branch it should at least build ;)

**Pull request merged**